pkg/proxy/iptables/proxier.go

```diff
@@ -986,8 +986,25 @@ func (proxier *Proxier) syncProxyRules() {
 					writeLine(proxier.natRules, append(args, "-j", string(KubeMarkDropChain))...)
 				}
 			}
+		} else {
+			// REJECT rules if there is no service endpoint
+			for _, ingress := range svcInfo.LoadBalancerStatus.Ingress {
+				if ingress.IP != "" {
+					// Add a reject rule so the behavior is the same for clients going via the load balancer
+					// (immediate reject instead of connection timeout). This also avoids a stale entry in the
+					// contrack table.
+					writeLine(proxier.filterRules,
+						"-A", string(kubeForwardChain),
```

Comment:
I think this wants to be `kubeServicesChain`.

Reply:
kubeServicesChain is called from OUTPUT. To prevent the default FORWARD we need to add the rule to the FORWARD chain; it has no effect in the KUBE-SERVICES chain. Not sure whether it would work or should be added to the raw-prerouting or mangele-prerouting chains.

Reply:
we fixed forwarding to be called in both cases - please rebase and retry in services

Reply:
This somehow contradicts the other comment. Sure, I will change this if the other comment is resolved.

Reply:
I don't understand what you mean - what other comment?  In My PR I linked Input to kubeServicesChain, too.